### ISSUE TITLE
The IN-predicate on "deliverycapacities.deliverydate" was invoked with an empty sequence.

### ISSUE THREAD

**Issue by kentbower** [kbower/tickettest1#2345]
_Imported from trac ticket 2345.
Created by **dwarner**
Opened in trac: May 21, 2013, 11:34:19 AM
Last modified in trac: May 21, 2013, 1:02:40 PM_
***
_by dwarner:_
```
Error - <class 'sqlalchemy.exc.SAWarning'>: The IN-predicate on
"deliverycapacities.deliverydate" was invoked with an empty 
sequence. This results in a contradiction, which nonetheless can 
be expensive to evaluate.  Consider alternative strategies for 
improved performance.
```

**Comment by kentbower** [comment]
_**kbower** trac comment on May 21, 2013, 1:02:40 PM:_

(Changeset [[4157]](../commit/c118f4baf86b1193dac50c50fdbbe19e63b14a2a))
* fixes #2345, The IN-predicate on "deliverycapacities.deliverydate" was invoked with an empty sequence. #stn

**Comment by kentbower** [comment]
_**kbower** trac status change on May 21, 2013, 1:02:40 PM:_ **closed**
